src/app/extensions/order-templates/shared/order-template-widget/order-template-widget.component.html

```diff
@@ -8,7 +8,9 @@
       <ng-container *ngIf="orderTemplates$ | async as orderTemplates; else emptyList">
         <ng-container *ngIf="orderTemplates.length; else emptyList">
           <div *ngFor="let orderTemplate of orderTemplates" class="mb-2">
-            {{ orderTemplate.title }}
+            <a [routerLink]="'/account/order-templates/' + orderTemplate.id" class="text-decoration-none">{{
+              orderTemplate.title
+            }}</a>
```

Comment:
This is not how we should create such links (even though we have done it in the past).
This is the way to go:
```
<a [routerLink]="['/account/order-templates', orderTemplate.id]" class="text-decoration-none">{{
      orderTemplate.title
}}</a>
```